### PR TITLE
Created Landing page for ecom.

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
What to review : 
Logic used for creating CategoryMen and CategoryWomen pages.
Nothing to be reviewed in LandingPage.

What not to review : 
CSS files are already reviewed.

### CODE REVIEW

File: src/pages/category-men/CategoryMen.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            <div className="filter flex ">
+                <p className="filter-heading margin-left margin-bottom margin-top">Filters</p>
+                <a className="clear-link margin-right margin-bottom margin-top" href="#">Clear</a>
+            </div>
+            <div className="filter">
+                <p className="filter-heading-text margin-left">Category</p>
+                <div className="flex">
+                    <input className="margin-left margin-top margin-bottom" id="category-checkbox1" name = "category"
+                        type="checkbox" checked={showCategoryMen}
+                        onChange={() => dispatch({ type: "CATEGORY_MEN" })}/>
```
Comment (ark2002) [inline src/pages/category-men/CategoryMen.js]: hey where is this dispatch declared that u are calling it here. cause u have used a useState and dispatch is usually used with a useReducer . so didnt quite from where is this dispatch getting called

File: src/pages/category-men/CategoryMen.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+const CategoryMen = () => {
+    const [categoryMenProducts , setCategoryMenProducts ] = useState([])
+    useEffect(() => {
+        axios.get("/api/products").then((response) => {
+            const result = response.data.products
+            setCategoryMenProducts(result.filter(product => product.categoryMen))
+        })
+      }, []);
+    return <div>
```
Comment (ark2002) [inline src/pages/category-men/CategoryMen.js]: use a parenthesis ( ) for your returns if it is gonna be a multiline

File: src/pages/category-men/CategoryMen.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                        type="checkbox" checked={showCategoryMen}
+                        onChange={() => dispatch({ type: "CATEGORY_MEN" })}/>
+                    <label className="input-label1 margin-bottom margin-top" for="category-checkbox1">
+                        Men Clothing
+                    </label>
+                </div>
+                <div className="flex">
+                    <input className="margin-left margin-bottom" id="category-checkbox2" name = "category"
+                        type="checkbox" checked={showCategoryWomen}
+                        onChange={() => dispatch({ type: "CATEGORY_WOMEN" })}  />
```
Comment (ark2002) [inline src/pages/category-men/CategoryMen.js]: here aswell

File: src/pages/category-men/CategoryMen.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+                 <i className="material-icons like-icon black"> favorite_border</i>
+ 
+                 <div className="wishlist-product-info">
+                     <p className="wishlist-info-text margin-top-bottom ">
+                         {product.name}
+                     </p>
+                     <p className="price-text margin-top-bottom "> Rs <span class = "line-through"> {product.originalPrice}</span> {product.discountedPrice}  </p> 
+                     <p className = "margin-top-bottom">Rating : {product.rating}</p>
+                 </div>
+                 <button onClick={ () =>addToCartHandler(product) } className="add-to-cart-btn primary-bg">Add to cart</button>
```
Comment (ark2002) [inline src/pages/category-men/CategoryMen.js:106]: this too

**Review by ark2002 — APPROVED** [review]
adding a preview to the pr would had helped in knowing the working of the logic better also there are some loopholes in the code with dispatches and cart handelers being called dont quite know where those are declared

File: src/pages/category-men/CategoryMen.js (hunk 4 above)
Comment (syed-shaida-hussain) [reply src/pages/category-men/CategoryMen.js:106]: Fixed the suggested changes.

**Review by yogesh- — APPROVED** [review]
Code is modular.
All suggested changes are done.
Can be merged.

pls add image/video/live url for further branches/features/pages

**Pull request merged**